config/bitcoin.go

```diff
@@ -12,27 +12,26 @@ import (
 
 // BTCConfig defines configuration for the Bitcoin client
 type BTCConfig struct {
-	DisableClientTLS  bool                      `mapstructure:"no-client-tls"`
-	CAFile            string                    `mapstructure:"ca-file"`
-	Endpoint          string                    `mapstructure:"endpoint"`
-	WalletEndpoint    string                    `mapstructure:"wallet-endpoint"`
-	WalletPassword    string                    `mapstructure:"wallet-password"`
-	WalletName        string                    `mapstructure:"wallet-name"`
-	WalletCAFile      string                    `mapstructure:"wallet-ca-file"`
-	WalletLockTime    int64                     `mapstructure:"wallet-lock-time"` // time duration in which the wallet remains unlocked, in seconds
-	TxFeeMin          chainfee.SatPerKVByte     `mapstructure:"tx-fee-min"`       // minimum tx fee, sat/kvb
-	TxFeeMax          chainfee.SatPerKVByte     `mapstructure:"tx-fee-max"`       // maximum tx fee, sat/kvb
-	DefaultFee        chainfee.SatPerKVByte     `mapstructure:"default-fee"`      // default BTC tx fee in case estimation fails, sat/kvb
-	EstimateMode      string                    `mapstructure:"estimate-mode"`    // the BTC tx fee estimate mode, which is only used by bitcoind, must be either ECONOMICAL or CONSERVATIVE
-	TargetBlockNum    int64                     `mapstructure:"target-block-num"` // this implies how soon the tx is estimated to be included in a block, e.g., 1 means the tx is estimated to be included in the next block
-	NetParams         string                    `mapstructure:"net-params"`
-	Username          string                    `mapstructure:"username"`
-	Password          string                    `mapstructure:"password"`
-	ReconnectAttempts int                       `mapstructure:"reconnect-attempts"`
-	BtcBackend        types.SupportedBtcBackend `mapstructure:"btc-backend"`
-	ZmqSeqEndpoint    string                    `mapstructure:"zmq-seq-endpoint"`
-	ZmqBlockEndpoint  string                    `mapstructure:"zmq-block-endpoint"`
-	ZmqTxEndpoint     string                    `mapstructure:"zmq-tx-endpoint"`
+	DisableClientTLS  bool                  `mapstructure:"no-client-tls"`
+	CAFile            string                `mapstructure:"ca-file"`
+	Endpoint          string                `mapstructure:"endpoint"`
+	WalletEndpoint    string                `mapstructure:"wallet-endpoint"`
+	WalletPassword    string                `mapstructure:"wallet-password"`
+	WalletName        string                `mapstructure:"wallet-name"`
+	WalletCAFile      string                `mapstructure:"wallet-ca-file"`
```

Comment:
This one is not needed

Reply:
Nice spot, but we do need `WalletPassword` and `WalletName` see [this](https://github.com/babylonlabs-io/vigilante/blob/137b7db70e5cdc3871210bb3a629f0ead8ab0c3f/btcclient/client_wallet.go#L34) and [this](https://github.com/babylonlabs-io/vigilante/blob/137b7db70e5cdc3871210bb3a629f0ead8ab0c3f/submitter/relayer/relayer.go#L225)